modules/mcuboot/boot/zephyr/Kconfig

```diff
@@ -15,8 +15,14 @@ partition-size=0x1e000
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_config"
 
 partition=MCUBOOT_PAD
-partition-size=0x200
+if SOC_PLATFORM_NRF54L
+	partition-size=0x800
```

Comment:
why 0x800?

Reply:
nRF54L15 has so big vector table.

Reply:
0x800 seems a bit many

Reply:
![image](https://github.com/nrfconnect/sdk-nrf/assets/56024351/25650f9b-feec-4e56-9d0e-ac89d446741f)
Gives 271 IRQS.
With
![image](https://github.com/nrfconnect/sdk-nrf/assets/56024351/c60f0e54-617c-4df6-8d7c-22ae9db41245)
so there is mandatory 32 words (4 bytes each that cover the base 16 interrupts), this takes 128 bytes already.
271-16 = 255 additional interrupt vectors, one word each is 4 * 255 = 1020, so 1148 bytes total.
This rounded up to next power of two is 2048, which is 0x800.
Am I correct here?

Reply:
Ref.: https://github.com/zephyrproject-rtos/zephyr/blob/main/cmake/linker_script/arm/linker.cmake#L153
Vector alignment is 0x800.